spec/elements/break_spec.rb

```diff
@@ -31,20 +17,11 @@
     end
   end
 
-  for_attribute(:type) do
-    with_value(:column) do
-      it_should_assign_successfully
-      it_should_output "<w:br w:type=\"column\"/>"
-    end
-
-    with_value(:page) do
-      it_should_assign_successfully
-      it_should_output "<w:br w:type=\"page\"/>"
-    end
-
-    with_value(:textWrapping) do
+  for_attribute(:type, with_namespace: :w) do
+    allowed = %i(column page textWrapping)
+    with_values(allowed) do
       it_should_assign_successfully
-      it_should_output "<w:br w:type=\"textWrapping\"/>"
+      it_should_output_regular_xml
```

Comment:
I think we made this change to streamline our test writing and to be able to test more exhaustively, especially with attributes that could take multiple different types of values. I'd agree that it's not as good for documentation, but I guess we fell down on the other side of the tradeoff. I can go back and refactor the tests to use the more verbose `it_should_output`.